src/responses.rs

```diff
@@ -158,38 +158,38 @@ pub struct DecryptedRoomEvent {
     #[wasm_bindgen(readonly)]
     pub event: JsString,
 
-    encryption_info: Option<EncryptionInfo>,
+    encryption_info: EncryptionInfo,
 }
 
 #[wasm_bindgen]
 impl DecryptedRoomEvent {
     /// The user ID of the event sender, note this is untrusted data
     /// unless the `verification_state` is as well trusted.
```

Comment:
If you can be bothered, it would be nice to re-word this and the similar docs in this file